config.json

```diff
@@ -2695,6 +2695,29 @@
           "integers"
         ],
         "difficulty": 3
+       },
+       {
+        "slug": "two-bucket",
+        "name": "Two Bucket",
+        "uuid": "07b4679a-3d9f-42bc-ad1e-b9ba272a1c15",
+        "prerequisites": [
+          "atoms",
+          "tuples",
+          "integers",
+          "lists",
+          "pattern-matching",
+          "if",
+          "cond",
+          "match",
+          "structs",
+          "enum",
+          "recursion"
+        ],
+        "practices": [
+          "atoms",
+          "integers"
+        ],
```

Comment:
Those are some pretty basic concepts for a rather complex exercise 🤔 and the check is failing for integers... maybe we could leave it completely empty? Or we say this is specifically a tail call recursion exercise and it practices and requires `tail-call-recursion`?

Reply:
This is not really a tail call recursion, it's more of a breadth first search algorithm. I don't remember why I wrote these two. I think it's best to leave empty.

Reply:
Oh I remember now, copy paste :(
Sorry about that!